homeassistant/components/schlage/select.py

```diff
@@ -0,0 +1,86 @@
+"""Platform for Schlage select integration."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.select import SelectEntity, SelectEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import SchlageDataUpdateCoordinator
+from .entity import SchlageEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class SchlageSelectEntityDescription(SelectEntityDescription):
```

Comment:
Do we need a custom description class? It's not adding anything is it?

Reply:
I can remove it. The other integrations used it